cirro/sdk/portal.py

```diff
@@ -138,3 +139,60 @@ def list_reference_types(self) -> DataPortalReferenceTypes:
                 for ref in self._client.references.get_types()
             ]
         )
+
+
+class DataPortalLogin:
+    """
+    Start the login process, obtaiing the authorization message from Cirro
+    needed to confirm the user identity.
+
+    Usage:
+
+        # Replace app.cirro.bio as appropriate
+        login = DataPortalLogin(base_url="app.cirro.bio")
+
+        # Present the user with the authorization message
+        print(login.auth_message)
+
+        # Generate the authenticated DataPortal object,
+        # blocking until the user completes the login process in their browser
+        portal = login.await_completion()
+    """
+    base_url: str
+    auth_info: DeviceCodeAuth
+
+    def __init__(self, base_url: str = None, enable_cache=False):
```

Comment:
I think in this case `enable_cache` should always be false so we can omit from the parameter.

Is there a case where you'd use this with cache?

Reply:
I can't imagine a case, but I also don't see a reason to omit it. Just the principle that we can't always imagine all the use cases. Unless there is a good reason to leave it out?

Reply:
Ok sounds good.